include/revng/Yield/CrossRelations/RelationDescription.h

```diff
@@ -14,10 +13,11 @@ type: struct
 fields:
   - name: Location
     type: string
-  - name: Related
+  - name: IsCalledFrom
     sequence:
       type: SortedVector
-      elementType: RelationTarget
+      elementType: string
+    optional: true
```

Comment:
This is a fixup over the previous change.

No fixup commits. Please don't do this, it makes my review longer and in the end I always have to ask you to drop them.

Reply:
Ah wait, I thought I squashed this one. It's my bad.